exercises/04_power/power.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import "fmt"
+
+func Pow(x, y uint) uint {
+	if y == 0 {
+		x = 1
+		return x
+	}
```

Comment:
optional: consider getting rid of this `if` block. You will need to do some changes to for-loop below